### PR TITLE
Add link to Elasticsearch docs from 7.4 What's New

### PR DESCRIPTION
**Pull request by achatterjee-grafana**
From the 7.4 What's New docs.

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-in-v7-4.md [hunk 1]
```diff
@@ -125,6 +125,8 @@ Grafana 7.4 includes the following enhancements
 
 > **Note:** We have deprecated browser access mode. It will be removed in a future release.
 
+For more information, refer to the [Elasticsearch docs]({{<relref "../datasources/elasticsearch.md">}}).
```
Comment (oddlittlebird) [inline docs/sources/whatsnew/whats-new-in-v7-4.md:128]: Per the style guide, link text should exactly match the title of the link target whenever possible. Please reconcile.

**Review by oddlittlebird — CHANGES_REQUESTED** [review]

File: docs/sources/whatsnew/whats-new-in-v7-4.md (hunk 1 above)
Comment (achatterjee-grafana) [reply docs/sources/whatsnew/whats-new-in-v7-4.md:128]: I see you have already made the change. Thank you.

Comment (oddlittlebird) [reply docs/sources/whatsnew/whats-new-in-v7-4.md:128]: I'm not seeing it in GitHub. Weird.

**Review by oddlittlebird — APPROVED** [review]
LGTM!

**Pull request merged**